db/pre-re/status.yml

```diff
@@ -415,6 +440,8 @@ Body:
     Flags:
       BossResist: true
       TaekwonAngel: true
+    EndReturn:
+      Curse: true
```

Comment:
There must be some hardcoded code that removes Curse by Blessing, you can remove that code now. (There is probably more code that can be removed because of this update.)

Reply:
Will resolve this in #6834.

Reply:
Shouldn't you undo your change here then?